pkg/webhooks/kafkatopic_validator_test.go

```diff
@@ -168,21 +303,11 @@ func TestValidateTopic(t *testing.T) {
 
 	topic.Spec.ReplicationFactor = 1
 
-	// Test overwrite attempt
 	err = kafkaClient.CreateTopic(&kafkaclient.CreateTopicOptions{Name: "test-topic", ReplicationFactor: 1, Partitions: 2})
 	if err != nil {
 		t.Error("creation of topic should have been successful")
 	}
 	topic.Name = "test-topic"
-	fieldErrorList, err = kafkaTopicValidator.validateKafkaTopic(context.Background(), topic, logr.Discard())
-	if err != nil {
-		t.Errorf("err should be nil, got: %s", err)
-	}
-	if len(fieldErrorList) != 1 {
-		t.Error("Expected not allowed due to existing topic with same name, got allowed")
-	} else if !strings.Contains(fieldErrorList.ToAggregate().Error(), "topic already exists on kafka cluster") {
-		t.Error("Expected not allowed for reason: already exists")
-	}
```

Comment:
**Question**: Why are we removing this section?

Reply:
I felt like it as a test duplicate because I wrote a better test for this here:  https://github.com/banzaicloud/koperator/blob/3444e5d819a4b4a9ac653307e498c6daf80bcf0c/pkg/webhooks/kafkatopic_validator_test.go#L75